optuna/samplers/_cmaes.py

```diff
@@ -89,6 +95,20 @@ def objective(trial):
 
             Note that the parameters of the first trial in a study are always sampled
             via an independent sampler, so no warning messages are emitted in this case.
+
+        consider_pruned_trials:
+            If this is :obj:`True`, the PRUNED trials are considered for sampling.
+
+            .. note::
+                Added in v2.0.0 as an experimental feature. The interface may change in newer
+                versions without prior notice. See
+                https://github.com/optuna/optuna/releases/tag/v2.0.0.
+
+            .. note::
+                It is suggested to set this flag :obj:`False` when the `MedianPruner` is used.
+                On the other hand, it is suggested to set this flag :obj:`True` when the
+                `HyperbandPruner` is used. Please see `the benchmark result
```

Comment:
Nits but how about making proper links out of "MedianPruner" and "HyperbandPruner"?